### PR TITLE
Android ACS 2.0.3 release notes

### PR DESCRIPTION
**Pull request by rymorale**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/documentation/adobe-campaign-standard/release-notes.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## February 17, 2023
 
+### Android Campaign Standard 2.0.3
```
Comment (swarna04) [inline src/pages/documentation/adobe-campaign-standard/release-notes.md:5]: Incorrectly placed under February 17 notes, let's fix it.

File: src/pages/documentation/release-notes/index.md [hunk 2]
```diff
@@ -4,6 +4,12 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## May 4, 2023
+
+### Android Campaign Standard 2.0.3
+
+* Adds support for url encoded URL's with query parameters to be used as an in-app message clickthrough URL.
```
Comment (swarna04) [inline src/pages/documentation/release-notes/index.md:11]: How about this?
```
Adds support for URL-encoded URLs with query parameters used to configure in-app message clickthrough destination.
```

**Review by swarna04 — CHANGES_REQUESTED** [review]

File: src/pages/documentation/release-notes/index.md (hunk 2 above)
Comment (rymorale) [reply src/pages/documentation/release-notes/index.md:11]: that sounds better. will update it.

**Review by calebk1 — APPROVED** [review]
LGTM!

**Pull request merged**